### PR TITLE
Add helper convert to nquads

### PR DESCRIPTION
**Pull request by vineetpant**
## Description

This MR adds `helper_convert_credential_to_nquads` helper function to Credentials helper to provide support for nquads conversion to sdk functions

## Details

- Add `helper_convert_credential_to_nquads`
- Update bindings for c, wasm and cli
- Add test case for `helper_convert_credential_to_nquads`
- Update Versions.md

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -254,6 +254,17 @@ async fn main() -> Result<()> {
                     .await?
             }
             #[cfg(all(feature = "vc-zkp-bbs", feature = "did-sidetree"))]
```
Comment (wulfraem) [inline src/main.rs]: Do we need `did-sidetree` for the nquads? ^^

File: src/api/vade_evan_api.rs [hunk 2]
```diff
@@ -496,6 +496,84 @@ impl VadeEvan {
             .map_err(|err| err.into())
     }
 
+    /// Converts a given credential to nquads Vector
```
Comment (wulfraem) [inline src/api/vade_evan_api.rs]: I think we should mention in the comments that we create canonized nquads as they're slightly more specific than normal nquads.

**Review by wulfraem — CHANGES_REQUESTED** [review]

File: src/main.rs (hunk 1 above)
Comment (vineetpant) [reply src/main.rs]: not required , removed it from all bindings and cli now

File: src/api/vade_evan_api.rs (hunk 2 above)
Comment (vineetpant) [reply src/api/vade_evan_api.rs]: updated comments

**Review by wulfraem — APPROVED** [review]

**Review by S3bb1 — APPROVED** [review]

**Pull request merged**